match2/commons/match_group_display_matchlist.lua

```diff
@@ -136,8 +142,8 @@ function MatchlistDisplay.Match(props)
 
 	-- Compute widths of the 2 opponent and 2 score columns. The small offsets
 	-- are due to border splitting from table-layout: auto.
-	local opponentWidth = math.floor(0.4 * props.width) - 1
-	local scoreWidth = 0.5 * (props.width - 5 - 2 * opponentWidth)
+	local scoreWidth = math.max(30, math.floor(0.1 * props.width) - 1)
+	local opponentWidth = 0.5 * (props.width - 5 - 2 * scoreWidth)
```

Comment:
Lotta magic numbers here, it would be nice if at least the `30` was put into a descriptive constant.